### DISCUSSION TITLE
非常好的愚人节更新，使我的大脑旋转~

### DISCUSSION THREAD

**Discussion by yxc0915** [QuickShop-Community/QuickShop-Hikari#1484] [General]
非常好的愚人节更新，使我的大脑旋转~
![_}9(F8H1$YA6E`05`O6~RFO](https://github.com/Ghost-chu/QuickShop-Hikari/assets/62410385/e0df0df5-07ca-49a7-927c-9ff36c82aa0e)

**Comment by Ghost-chu** [comment]
链接部分还是搞砸了（）  
去年就应该修来着，结果忘记了整整一年x

**Reply by yxc0915** [reply]
很好，他修改了
![image](https://github.com/Ghost-chu/QuickShop-Hikari/assets/62410385/5104dd22-3678-442a-95cd-6c5b894c229b)
